### PR TITLE
Link to the discourse forum

### PR DESCRIPTION
**Pull request by zuphilip**

### CODE REVIEW

**Comment by rmzelle** [comment]
Yes, I realized I hadn't updated the site yet. Will take a look soon.

**Comment by zuphilip** [comment]
@rmzelle What is the status here?

File: _pages/developers.md [hunk 1]
```diff
@@ -161,3 +161,5 @@ Still, many CSL contributors are Zotero users, and the lively [Zotero forums](ht
 Prior to the move to GitHub, from late 2009 to March 2011, the project code was maintained in Mercurial repositories on [Bitbucket.org](https://bitbucket.org/) (see the mostly retired [Bitbucket repositories of Bruce D'Arcus](https://bitbucket.org/bdarcus)), and before that, from February 2006 to late 2009, in an SVN repository on SourceForge (see the [xbiblio SourceForge project page](http://sourceforge.net/projects/xbiblio/)).
 Before migrating to GitHub during late 2010/early 2011, CSL styles were separately maintained in the now retired [Zotero SVN](https://www.zotero.org/trac).
 The [xbiblio-devel mailing list](https://lists.sourceforge.net/lists/listinfo/xbiblio-devel), hosted by SourceForge, has been in continued use since 2004.
```
Comment (adam3smith) [inline _pages/developers.md]:
```suggestion

```

we'd want this deleted too

**Comment by adam3smith** [comment]
Looks good to me with the one comment I added.

**Comment by zuphilip** [comment]
Interesting, this new suggestion feature in Github. Your suggestion is now part of this PR.

**Comment by rmzelle** [comment]
Thanks!

> @rmzelle What is the status here?

I was originally hoping to fix up the imported posts from xbiblio-devel by hand (and e.g. properly escape all code excerpts), but I haven't even gotten started on that yet, so no reason to wait with merging this. Sorry for the delay!

**Pull request merged**